src/components/join/How.jsx

```diff
@@ -1,7 +1,75 @@
+"use client";
+import Image from "next/image";
+import image1 from "@/public/join/how.webp";
+import Link from "next/link";
+import useView from "../useView";
+
 const How = () => {
   return (
-    <div className="">
-      <p>How</p>
+    <div className="ml-7">
+      <p
+        className={`${
+          useView && "animate-fade-right animate-delay-[100ms]"
+        }text-2xl md:text-4xl text-swim-blue-400 font-bold mb-10 `}
+      >
+        How To Practice and Officially become a member:
+      </p>
+      <div className="md:flex md:flex-row">
+        <div
+          className={`${
+            useView && "animate-fade-right animate-delay-[150ms]"
+          } md:flex md:flex-col items-center`}
+        >
+          <Image
+            src={image1}
+            width={300}
+            height={300}
+            alt="How image instructions"
+          />
+          <p
+            className={`${useView && "animate-fade-right animate-delay-[300ms]"}
+            text-1.5xl mt-5 mb-5`}
+          >
+            2. Register for Swim Club on{" "}
+            <Link
+              className="text-swim-blue-300 underline mr-2"
+              href="https://highlanderlink.ucr.edu/organization/swimclubatucr"
+              target="_blank"
+            >
+              Highlander Link
+            </Link>
+            <br />
```

Comment:
instead of break tags, you can just use divider tags for the alphabetical list and use padding instead of `&emsp;`